Rubberduck.Core/Refactorings/MoveCloserToUsage/MoveCloserToUsageRefactoring.cs

```diff
@@ -168,9 +169,12 @@ private void InsertNewDeclaration()
             }
 
             var insertionIndex = (expression as ParserRuleContext).Start.TokenIndex;
+            var firstReferenceLine = _vbe.ActiveCodePane.CodeModule.GetLines((expression as ParserRuleContext).Start.Line, 1);
+            var indentLength = firstReferenceLine.Length - firstReferenceLine.TrimStart().Length;
+            var padding = new string(' ', indentLength);
```

Comment:
I observe that none of unit tests account for the case where there is something that isn't "indentable" -- such as line numbers for example. Thus, what happens when we have code like `100:     Foo = bar`? The assumption here is that we only indent if there is only whitespace on the line and that might be actually fine but the unit tests should prove that assumption.

Reply:
Very good point, but IMO not a showstopper - let's get the wrappers into `using` blocks, and if the line number stuff can't be fixed right away I'd still merge it.

Reply:
I've added a new test and it moves the line with the line number down without introducing any indent before the number so I think that's okay.

However, I tried a different test where the declaration has a line number. The declaration gets moved but leaves the line number behind. This is a problem when the line after the declaration has a line number too and suddenly we have a line with two line numbers! This is unrelated to the changes I made, so I will open a new issue.